tests/models/registry.py

```diff
@@ -270,6 +270,7 @@ def check_available_online(
                                         trust_remote_code=True),
     "NVLM_D": _HfExamplesInfo("nvidia/NVLM-D-72B",
                               trust_remote_code=True),
+    "OvisForConditionalGeneration": _HfExamplesInfo("AIDC-AI/Ovis1.6-Gemma2-9B",trust_remote_code=True),  # noqa: E501
```

Comment:
```suggestion
    "OvisForConditionalGeneration": _HfExamplesInfo("AIDC-AI/Ovis1.6-Llama3.2-3B",trust_remote_code=True),  # noqa: E501
```
We can use a smaller model for registry and test: [AIDC-AI/Ovis1.6-Llama3.2-3B](https://huggingface.co/AIDC-AI/Ovis1.6-Llama3.2-3B)